apps/registration/api.py

```diff
@@ -118,21 +102,16 @@ def post(self, request):
 
             # send account recovery email to user
             current_site = get_current_site(request)
-            message = render_to_string('acc_recover_email.html', {
-                'user': user,
-                'domain': current_site.domain,
-                'uid': urlsafe_base64_encode(force_bytes(user.pk)),
-                'token': account_activation_token.make_token(user),
-            })
-            mail_subject = 'Password reset request for your account.'
-            to_email = user.email
-            email = EmailMultiAlternatives(
-                mail_subject,
-                message,
-                to=[to_email]
-                )
-            email.attach_alternative(message, "text/html")
-            email.send()
+            send_template_email(
+                [user.email],
+                'Password reset request for your account.',
+                'acc_recover_email.html',
+                {
+                    'user': user,
+                    'domain': current_site.domain,
+                    'uid': urlsafe_base64_encode(force_bytes(user.pk)),
+                    'token': account_activation_token.make_token(user),
+                })
```

Comment:
nit: end last argument with a comma and have function call on its own line, like
```
    },
)
```